src/formatDate.js

```diff
@@ -51,6 +51,26 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const arrDate = date.split(fromFormat[3]);
+
+  const objDate = {};
+
+  for (let i = 0; i < 3; i++) {
+    objDate[fromFormat[i]] = arrDate[i];
+  }
+
+  objDate['YY'] 
+  ? 
+  objDate['YYYY'] = objDate['YY'] >= 30 ? 19 + objDate['YY'] : 20 + objDate['YY'] 
+  : objDate['YY'] = objDate['YYYY'].substring(2);
```

Comment:
```suggestion
  if (objDate['YY']) {
    if (objDate['YY'] >= 30) {
      objDate['YYYY'] = 19 + objDate['YY']
    } else {
      objDate['YYYY'] = 20 + objDate['YY'];
    }
  } else {
    objDate['YY'] = objDate['YYYY'].substring(2)
  }
```
В сложных случаях лучше не использовать тернарный оператор, а пользоваться if else